go/cache/ristretto/bloom/bbloom_test.go

```diff
@@ -38,7 +41,7 @@ func TestM_NumberOfWrongs(t *testing.T) {
 			cnt++
 		}
 	}
-	fmt.Printf("Bloomfilter New(7* 2**16, 7) (-> size=%v bit): \n            Check for 'false positives': %v wrong positive 'Has' results on 2**16 entries => %v %%\n", len(bf.bitset)<<6, cnt, float64(cnt)/float64(n))
+	log.Infof("Bloomfilter New(7* 2**16, 7) (-> size=%v bit): \n            Check for 'false positives': %v wrong positive 'Has' results on 2**16 entries => %v %%", len(bf.bitset)<<6, cnt, float64(cnt)/float64(n))
```

Comment:
This relates to all of the log changes -- `log.Infof` will not show up in the test/CI output. Is that what we want?

Reply:
Do we want the fmt.Printf()s? We don't do this in any other test. We can add it back if it is needed.